### ISSUE TITLE
what is the meaning of valid_category_weight 

### ISSUE THREAD

**Issue by wonshy** [OpenDriveLab/PersFormer_3DLane#93]
https://github.com/OpenPerceptionX/PersFormer_3DLane/blob/8d70aa431d9acd67484310299c2208bf98a0ef6c/models/networks/Loss_crit.py#L252

`valid_category_weight = torch.sum(gt_category_onehot[:, :, :, 1:], dim=-1).unsqueeze(-1)
`
Why gt_category_onehot  start adding from 1 instead of 0 ？

How should I understand this  valid_category_weight ？

**Comment by ilnehc** [comment]
The first position stands for invalid. `valid_categroy_weight` includes all valid lanelines to calculate other losses.
